### PR TITLE
feat(flashing port): allow pio to autodetect port if not specified

### PR DESCRIPTION
**Pull request by mtfurlan**
Also cleaned up a how we print stuff, so it's easier to read

### CODE REVIEW

File: espq.py [hunk 1]
```diff
@@ -115,6 +115,7 @@ def __init__(self, device):
         self.flashed = False
         self.online = False
         self.reported = {}
+        self.ip_addr = None
```
Comment (mkfink) [inline espq.py]: Did you mean to force it to override the IP listed in the config file?

**Review by mkfink — COMMENTED** [review]
Is it overriding the IP defined in the config file with None on purpose? That will cause issues if you want to change a device's topic.

File: espq.py (hunk 1 above)
Comment (mtfurlan) [reply espq.py]: Oh, no. I thought that happened somewhere else.

Comment (mtfurlan) [reply espq.py]: Better now?

Comment (mkfink) [reply espq.py]: Yeah, looks good to merge

**Pull request merged**